### PR TITLE
[nyx] Move nyx patches to standalone files

### PR DESCRIPTION
**Pull request by pyoor**
Whitespace in inlined patches can by modified by linters.  These should exist as standalone files.

### CODE REVIEW

File: services/nyx/patches/hongfuzz-2b-chunked.diff [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
++++ b/include/config.h
+@@ -459,7 +459,7 @@
+    problems with complex programs). You need to recompile the target binary
+    after changing this - otherwise, SEGVs may ensue. */
+
+-#define MAP_SIZE_POW2 16
++#define MAP_SIZE_POW2 23
+
+ /* Do not change this unless you really know what you are doing. */
+
```
Comment (jschwartzentruber) [inline services/nyx/patches/hongfuzz-2b-chunked.diff:190]: This chunk is unrelated to the honggfuzz mutator

File: services/nyx/build_afl.sh [hunk 2]
```diff
@@ -57,66 +57,23 @@ mkdir -p /srv/repos
...
-
--        if params.qemu_id == 0{
--            println!("[!] libnyx: spawning qemu with:\n {}", params.cmd.join(" "));
--        }
-+        println!("[!] libnyx: spawning qemu with:\n {}", params.cmd.join(" "));
-
-         let (shm_work_dir, file_lock) = Self::create_shm_work_dir();
-         let mut shm_work_dir_path = PathBuf::from(&shm_work_dir);
-EOF
+git ls-tree HEAD fuzz_runner/src/nyx/qemu_process.rs
```
Comment (jschwartzentruber) [inline services/nyx/build_afl.sh:73]: Do we need `ls-tree` here? It's fine for debugging, but then it's weird to only have this one file and not all patched files.

**Review by jschwartzentruber — CHANGES_REQUESTED** [review]

File: services/nyx/build_afl.sh (hunk 2 above)
Comment (pyoor) [reply services/nyx/build_afl.sh:73]: Nope.  This was left over from testing.

File: services/nyx/patches/hongfuzz-2b-chunked.diff (hunk 1 above)
Comment (pyoor) [reply services/nyx/patches/hongfuzz-2b-chunked.diff]: Moved to increase-map-size.diff.

**Review by jschwartzentruber — APPROVED** [review]
